src/pages/Groups/Group4/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,134 @@
+import styled from 'styled-components'
+
+export const ContainerDiv = styled.div`
+  background-color: '#ffffff';
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const LoginContainer = styled.div`
+  height: 100vh;
+  width: 90vw;
+  margin: 0px 23px;
+  padding-top: 50px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
+export const InputTextLabel = styled.text`
+  float: left;
+  font-size: 17px;
+  padding: 0px 0px 4px 0px;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 200;
+  margin-right: auto;
+`
+
+export const Logo = styled.img`
+  width: 70%;
+  max-width: 255px;
+  max-height: 255px;
+`
+export const ForgetPwButton = styled.div`
+  text-align: left;
+  text-decoration: underline;
+  color: #72ba75;
+  font-size: 13px;
+  margin-right: auto;
+  .ant-btn {
+    letter-spacing: 0;
+    text-align: center;
+    font-weight: 200;
+    color: #72ba75;
+  }
+  .ant-btn-primary {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    width: 100%;
+    margin-top: 10px;
+  }
+  .ant-btn-primary:focus {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+    font-weight: bold;
+  }
+  .ant-btn-primary:hover {
+    background-color: #72ba75;
+    border-color: #72ba75;
+    background: #72ba75;
+    border-color: #72ba75;
+    font-weight: bold;
+  }
+  .ant-btn-default: {
+    height: min-content;
+    margin: 10px;
+    float: left;
+    border-radius: 20px;
+  }
+`
+export const PostButton = styled.div`
```

Comment:
Also, whats a PostButton? Seems like a button container instead? though im not too sure if it works as intended